### DISCUSSION TITLE
Create issues related to PRs?

### DISCUSSION THREAD

**Discussion by brunoborges** [microsoft/gctoolkit#109] [General]
If a bug is introduced in a PR, does it really make sense to create an issue?
I consider issues as problems identified in a branch or tag, not in a PR.

I believe that If a PR introduces bugs/issues, these problems should be addressed, discussed, and fixed, within the PR itself.

**Comment by dsgrieve** [comment]
If the PR was merged and the bug was found after the merge, this absolutely merits creating a new issue. 

If the PR has not merged, then most of the time the problem should be resolved within the PR itself. Especially in the case of a regression. But there will always be cases where merging the PR is more urgent than fixing the uncovered bug/issue.
